doc/source/tutorials/bipartite_matching_maxflow/bipartite_matching_maxflow.rst

```diff
@@ -0,0 +1,89 @@
+.. include:: include/global.rst
+
+.. tutorials-bipartite-matching-maxflow
+
+==========================================
+Maximum Bipartite Matching by Maximum Flow
+==========================================
+
+This example presents how to visualise bipartite matching using maximum flow. Please note that the *igraph* already has :meth:`maximum_bipartite_matching` which is better suited for finding the maximum bipartite matching. For a demonstration on how to use that method instead, check out `Maximum Bipartite Matching<>`_. This particular example is purely for demonstrative purposes.
+
+.. TODO: add link to Maximum Bipartite Matching
```

Comment:
Remove the TODO comment if the link is already there (it seems like it is)

Reply:
Currently the link doesn't actually link anywhere which is why I've left that TODO there. I'll add a link when I have a better idea of the site structure.